src/providers/generic/game/directory_resolver/win/XGPDirectoryResolver.ts

```diff
@@ -0,0 +1,39 @@
+import GameDirectoryResolverProvider from '../../../../ror2/game/GameDirectoryResolverProvider';
+import Game from '../../../../../model/game/Game';
+import R2Error from '../../../../../model/errors/R2Error';
+import ManagerSettings from '../../../../../r2mm/manager/ManagerSettings';
+import FsProvider from '../../../../../providers/generic/file/FsProvider';
+import child from 'child_process';
+
+export default class XGPDirectoryResolver extends GameDirectoryResolverProvider {
```

Comment:
Minor but I'd rather this be renamed to XboxGamePassDirectoryResolver

The reasoning is that we don't really benefit from condensing the name. It's okay to be a little verbose as it helps understand the context. A random person going through the source code may not know what XGP means.